guides/assets/stylesheets/syntaxhighlighter/shCore.css

```diff
@@ -33,7 +33,7 @@
   height: auto !important;
   left: auto !important;
   line-height: 1.1em !important;
-  margin: 0 !important;
+  margin: 0.5px 0 0 0 !important;
```

Comment:
this is changing margin-top but the commit message says margin-bottom. I'd guess we want to change the bottom margin looking at screenshots on the linked issue.